### ISSUE TITLE
`master` should always work

### ISSUE THREAD

**Issue by Mierdin** [Juniper/contrail-ansible-deployer#15]
I noticed that the setup guide on https://github.com/tungstenfabric/website/wiki/Tungsten-Fabric:-10-minute-deployment-with-k8s-on-AWS refers to https://github.com/Juniper/contrail-ansible-deployer/commit/ceb29b0b2eb0f63409cf6098176258d793b84736 as being a "known working" commit.

Indeed, I ran into issues when choosing `master` instead, so it seems that in order to make use of these playbooks without running into problems, one must know which commit to use.

This is not a big deal if someone comes across these playbooks by means of this walkthrough, but hardly helpful for those who find it through other means. To cover the bases, we should ensure `master` is always in a working state.

Is there a current effort around ensuring this? Are there specific tasks that the community can help with to move towards this goal?

**Comment by Mierdin** [comment]
@Andrey-mp Thanks for the reply but I assumed there was some CI in place. My point is that it's not working. :smile:

Again, I'm happy to contribute in this capacity, but I wanted to get up to speed with the state of things in this area, or at the very least, raise the flag that `master` appears to not work.

**Comment by Mierdin** [comment]
*bump* - I keep hearing that eyes are starting to turn to this repo as the way forward for installing tungsten, and I think it's in everyone's best interest to make sure `master` is always working, so we don't turn people away at the door.

Anyone have any initiatives or plans in place today to get `master` back into a working state?

**Comment by Andrey-mp** [comment]
@Mierdin can you please describe the issue?
several CI-s check every commit into master. Changes can be merged only if they are checked successfully by CI.

If some specific case doesn't work - please file detailed bug.

**Comment by Mierdin** [comment]
I think I wasn't being clear. It depends on when I use these playbooks. Sometimes it works, sometimes it doesn't. Currently, they work. In the past, when I ran into issues, I went poking around and noticed there were guides referencing specific commits, rather than simply using `master`. This implied to me that either there was no CI, or that the CI in place wasn't sufficient.

Since you've said there's CI in place, I was just trying to find out more information about it, and how I can help out. Rather than drag this out, I'll close this issue and re-open when I encounter more issues, but I would just like to mention we should strive to encourage users **always** to use `master`, and not a "known working" commit.
